.htmllintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "plugins": [],
+
+  "id-class-style": "dash",
+  "indent-style": "spaces",
+  "indent-width": 2,
+  "attr-name-ignore-regex": "http-equiv"
```

Comment:
(After https://github.com/mozilla/testpilot-containers/pull/110/files#r98092302 is addressed), this needs to be `"attr-name-ignore-regex": "http-equiv|data.*"` or we get:
```
webextension/popup.html: line 116, col 63, attribute names must match the format: lowercase
webextension/popup.html: line 116, col 91, attribute names must match the format: lowercase
webextension/popup.html: line 117, col 63, attribute names must match the format: lowercase
webextension/popup.html: line 117, col 91, attribute names must match the format: lowercase
webextension/popup.html: line 118, col 63, attribute names must match the format: lowercase
webextension/popup.html: line 118, col 91, attribute names must match the format: lowercase
webextension/popup.html: line 119, col 63, attribute names must match the format: lowercase
webextension/popup.html: line 119, col 91, attribute names must match the format: lowercase
webextension/popup.html: line 120, col 63, attribute names must match the format: lowercase
webextension/popup.html: line 120, col 91, attribute names must match the format: lowercase
webextension/popup.html: line 121, col 63, attribute names must match the format: lowercase
webextension/popup.html: line 121, col 91, attribute names must match the format: lowercase
webextension/popup.html: line 122, col 63, attribute names must match the format: lowercase
webextension/popup.html: line 122, col 91, attribute names must match the format: lowercase
webextension/popup.html: line 123, col 63, attribute names must match the format: lowercase
webextension/popup.html: line 123, col 91, attribute names must match the format: lowercase
```